src/api/controllers/IdentityController.js

```diff
@@ -110,6 +110,65 @@ function getHashedMessage(identity, owner, type) {
     return hashedMessage;
 }
 
+/**
+ * Looks for the user identity.
+ * @returns {Promise<string|null>} The identity or `null` if none is found.
+ *
+ * Identity can be:
+ *   - a Solana (SNS) domain (`.sol`)
+ *   - an Ethereum (ENS) domain (`.eth`)
+ *   - a Point identity (registered in Point Identity Contract)
+ *
+ * The priority to choose an identity (should multiple be available) is:
+ *   - SNS
+ *   - ENS
+ *   - Point
```

Comment:
With this approach (looking first in SNS, then ENS and lastly Point), if an existing Point Identity happens to own an SNS or ENS domain with the same address, the next time the user access Point Network, they'll see their identity changed to the SNS or ENS domain.

How do we plan to handle these cases?

For example, this is the wallet page of an existing Point Identity `ynet_dell.point`, that also owns `pointnetwork.eth`:
![wallet_existing_identity](https://user-images.githubusercontent.com/101118664/181097521-c85f6cff-8259-4479-92c1-1c96e1ddaa19.png)

Reply:
Thanks @germanviescas, in order to send funds we use address and chain id, so no problem here.

In the UI for each network we can show the appropriate handle or the address relevant to the chain if the handle is not registered.

So let's say you have an address like `0x123...` and a corresponding Solana address `4567...`, which is the owner of `ynet_foo.point` identity and `foo.sol` Solana devnet domain respectively. In the wallet we should see:

| Currency   |      Address      |  Balance |
|----------|:-------------:|------:|
| Point |  ynet_foo.point | ... |
| yPoint |    ynet_foo.point   |   ... |
| rinkeby | 0x123... |    ... |
| devSol | foo.sol |    ... |
| Sol | 4567... |    ... |

In other words, show handles only for those chains where they exist, and the addresses for the rest ones. Does this answer your question?